# 3-11 Intentional Error
numbers = [
    '1',
    '2',
    '3'
]

print(numbers[3]) # this will cause famous list index out of range error